flask_resources/serializers/json.py

```diff
@@ -72,6 +73,55 @@ def serialize_object_list(self, obj_list):
         return json.dumps(obj_list, cls=self.encoder, **self.dumps_options)
 
 
+class CustomJSONSerializer:
```

Comment:
To me this is not a JSON serializer: this is a serializer to serialize an obj into a defined schema. The format can vary.

It should probably not go into a file `json.py`, but maybe in the `__init__.py`? Or a base?

And about the name, what about:
```suggestion
class MarshmallowSerializer:
```
?
Inspired from the class below.